packages/experience/src/apis/experience.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  type IdentificationApiPayload,
+  InteractionEvent,
+  type PasswordVerificationPayload,
+  SignInIdentifier,
+  type UpdateProfileApiPayload,
+} from '@logto/schemas';
+
+import api from './api';
+
+const prefix = '/api/experience';
+
+const experienceRoutes = Object.freeze({
+  prefix,
+  identification: `${prefix}/identification`,
+  verification: `${prefix}/verification`,
+  profile: `${prefix}/profile`,
+  mfa: `${prefix}/profile/mfa`,
+});
+
+type VerificationResponse = {
+  verificationId: string;
+};
+
+type SubmitInteractionResponse = {
+  redirectTo: string;
+};
+
+const initInteraction = async (interactionEvent: InteractionEvent) =>
+  api.put(`${experienceRoutes.prefix}`, {
+    json: {
+      interactionEvent,
+    },
+  });
+
+const identifyUser = async (payload: IdentificationApiPayload = {}) =>
+  api.post(experienceRoutes.identification, { json: payload });
+
+const submitInteraction = async () =>
+  api.post(`${experienceRoutes.prefix}/submit`).json<SubmitInteractionResponse>();
```

Comment:
```suggestion
  api.post(`${experienceRoutes.submit}`).json<SubmitInteractionResponse>();
```
add `submit` to the constant?